OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3707,66 +3710,93 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                
+
                                 <item xml:id="e12">
                                     <locus target="#1r #44v #69v #96v #123v"/>
                                     <desc>Stamps.</desc> <!--I cannot read the content of the stamps-->
                                 </item>
                             </list>
                         </additions>
-                        
+
                         <bindingDesc>
                             <binding xml:id="binding" contemporary="true">
-                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards covered with reddish-brown blind-tooled <material
-                                        key="leather">leather</material>. The front and back board are decorated with red velvet inlays.</decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                                    <material key="wood"/>
-                                    <material key="leather"/>
-                                    <material key="textile"/>
+                                    <decoNote xml:id="b1" type="bindingMaterial">
+                                      The book is bound in <material key="wood">wooden</material> boards covered with reddish-brown blind-tooled <material
+                                      key="leather">leather</material>. The front and back board are decorated with red <material key="textile">velvet</material> inlays.</decoNote>
+                                      <decoNote xml:id="b2" type="Boards">
+                                      The boards are cut to the same size as the bookblock and present external shallow bevels.
+                                    </decoNote>
+                                    <decoNote xml:id="b3" type="Cover" color="reddish brown">
+                                      Full reddish brown leather cover.
+                                      The turn-ins have been trimmed out and at the corners they form butt mitres.
+                                      An <term key="additionalLeatherPatch"></term> has been added along the inner margin of the boards to fill the gap between the turn-ins.
+                                      The cover shows a fine blind-tooled decoration, identical on both sides of the cover. The turn-ins and the edges are decorated as well.
+                                      The decoration consists in seven concentric frames surrounding a central rectangular panel featuring a Latin cross.
+                                      Frames composed by the repetition of single tools, enclosed between <term key="tripleStraightLine">triple straight lines</term>,
+                                      alternate with empty ones.
+                                      Proceeding from the outer to the inner frame, the repeating decoration motifs are: <term key="crissCross"></term> (two variants),
```

Comment:
Perhaps the colon is unneeded in this position, before a list. @gnizla ?

Reply:
You are right, Grammarly suggests to remove the colon :) ready to modify it if @gnizla has another opinion